batch/batch/server/server.py

```diff
@@ -270,6 +270,7 @@ def to_json(self):
 
 app = Flask('batch')
 
+log.info(f'app.root_path = {app.root_path}')
```

Comment:
Should this be here?

Reply:
Yes.  It was extremely useful piece of debugging that I'd like to keep around.